recipe/meta.yaml

```diff
@@ -1,17 +1,17 @@
 {% set name = "omegaconf" %}
-{% set version = "1.4.1" %}
+{% set version = "2.0.0rc24" %}
```

Comment:
I released rc25 today and will likely release rc26 tomorrow.

Reply:
I'll bump to the latest one before merging.